packages/forms/src/elements/common/Message.tsx

```diff
@@ -42,8 +54,8 @@ export const Message = React.forwardRef<HTMLDivElement, IMessageProps>(
 
     let combinedProps = { validation, validationLabel, ...props };
 
-    if (fieldContext) {
-      combinedProps = fieldContext.getMessageProps(combinedProps);
+    if (typeof getMessageProps === 'function') {
```

Comment:
This seems indicative of poor typing within the context, but might not be worth addressing here.

Reply:
I've cleaned this up and removed the `typeof` check, I was mistakenly thinking the Field context could be misused (however it is not publicly exported).